bindings/rust/standard/integration/src/network/tls_client.rs

```diff
@@ -46,38 +46,17 @@ mod kms_pq {
     // supports ML-KEM.
     #[test_log::test(tokio::test)]
     async fn pq_handshake() -> Result<(), Box<dyn std::error::Error>> {
-        let policy = Policy::from_version("KMS-PQ-TLS-1-0-2020-07")?;
+        let policy = Policy::from_version("PQ-TLS-1-2-2023-10-09")?;
         let tls = handshake_with_domain(DOMAIN, &policy).await?;
 
         assert_eq!(
             tls.as_ref().cipher_suite()?,
-            "ECDHE-KYBER-RSA-AES256-GCM-SHA384"
+            "TLS_AES_256_GCM_SHA384"
         );
-        assert_eq!(tls.as_ref().kem_name(), Some("kyber512r3"));
+        assert_eq!(tls.as_ref().kem_group_name(), Some("x25519_kyber-512-r3"));
 
         Ok(())
     }
-
-    // We want to confirm that non-supported kyber drafts successfully fall
-    // back to a full handshake.
```

Comment:
Why are we removing this test? We can't just update it to use TLS1.3?

Reply:
This test is testing that all the Round 1 PQ TLS 1.2 Policies (that previously contained only BIKE/SIKE support) fall back from `ECDHE-BIKE-RSA-AES256-GCM-SHA384` / `ECDHE-SIKE-RSA-AES256-GCM-SHA384` to the classical `ECDHE-RSA-AES256-GCM-SHA384` cipher now that BIKE/SIKE support has been removed from KMS endpoints. Those BIKE/SIKE ciphers have already been removed from these s2n Security Policies long ago. Since these tests are specifically testing PQ TLS 1.2 fallback behavior, they can't be migrated to PQ TLS 1.3.

Reply:
Gotcha, so the old test was more about making extra sure that BIKE/SIKE were impossible than testing a fallback mechanism (bc TLS1.3 also has a fallback mechanism: with a HRR a PQ policy can negotiate non-PQ). We could add a new version of the test that makes extra sure that TLS1.2 PQ is never negotiated, but that probably belongs in the C unit tests anyway.